pkg/database/database.go

```diff
@@ -94,12 +94,10 @@ func ConnectDB(
 				return conn.RegisterFunc("regexp", func(re, s string) bool {
 					result, ok := cache.Get(re)
 					if !ok {
-						complied, err := regexp.Compile(re)
+						result, err = regexp.Compile(re)
 						if err != nil {
 							return false
 						}
```

Comment:
we need to add the result to the cache!

Reply:
done.

Reply:
btw - I don't like how it looks now. much better to have another `return` statement.